mobile/src/main/java/org/openhab/habdroid/ui/preference/CustomInputTypePreference.kt

```diff
@@ -67,26 +69,52 @@ open class CustomInputTypePreference constructor(context: Context, attrs: Attrib
         return PrefFragment.newInstance(key, title, inputType, autofillHints)
     }
 
-    class PrefFragment : EditTextPreferenceDialogFragmentCompat() {
+    class PrefFragment : EditTextPreferenceDialogFragmentCompat(), TextWatcher {
+        private lateinit var wrapper: TextInputLayout
```

Comment:
This cannot be `lateinit`, because it isn't initialized below if `view == null`. Change it to
```suggestion
        private var wrapper: TextInputLayout? = null
```